libs/@hashintel/design-system/src/selector-autocomplete/selector-autocomplete-option.tsx

```diff
@@ -65,6 +73,8 @@ export const SelectorAutocompleteOption = ({
   const onMouseEnter = () => (imageUrl ? setShowPreviewPane(true) : null);
   const onMouseLeave = () => (imageUrl ? setShowPreviewPane(false) : null);
 
+  const typeTitle = slugToTitleCase(typeId.split("/").slice(-3, -2)[0]!);
```

Comment:
I think this will only work for types created in HASH, if we're rendering external types here hosted on an arbitrary URL we would need to lookup the title in the type itself. Not how soon we'll encounter this, but maybe worth leaving a `@todo` or just rendering the URL using the `OntologyChip` component when `slugToTitleCase` returns `""` here.

Reply:
Thanks, done in https://github.com/hashintel/hash/pull/3167/commits/e43ca7d4c7d46a877faef955b6e5e120bc765f9b